src/components/asset-list/RecyclerAssetList/index.tsx

```diff
@@ -260,7 +260,6 @@ function RecyclerAssetList({
   colors,
   sections,
   openInvestmentCards,
-  openFamilyTabs,
   openSavings,
```

Comment:
can we do similarly for `openSavings` `openInvestmentCards` and `openSmallBalances`

Reply:
I think we don't need this, because the only place where we're using this is inside showcase.